### PR TITLE
Adds a new traitor item: The Visicar Armblade

### PR DESCRIPTION
**Pull request by CheffieGithub**
## About the PR 
Adds a new traitor item for all jobs (or just miners). The Visicar a blade which attaches and detaches from your arm and only works while attached. Most of the information is in the design doc: https://hackmd.io/@Cheffie/BySba82Z2

## Why's this needed? 
Traitor item variety ! The 3 rampage items ( or 4 if redsaw is counted ) all have their quirks and uniqueness but we’ve all used them plenty, perhaps a new one would help mix things up especially with a different damage type than brute.

## Changelog 

```changelog
(u)Cheffie & Vispectra
(*)The syndicate have been spotted outfitting their agents with a new weapon
```

### CODE REVIEW

**Comment by Zonespace27** [comment]
image
you broke the rubber hammer sprite btw

**Comment by CheffieGithub** [comment]
screm

File: code/obj/item/misc_weapons.dm [hunk 1]
```diff
@@ -1927,3 +1927,159 @@ obj/item/whetstone
...
+	abilities = list(/obj/ability_button/armblade_toggle)
+
+	New()
+		..()
+		var/obj/item/mining_tool/T = new /obj/item/mining_tool(src)
+		src.tool = T
+		T.name = src.name
+		T.desc = src.desc
+		T.dig_strength = 5
+		T.hitsound_charged = 'sound/weapons/cutter.ogg'
```
Comment (virvatuli-gh) [inline code/obj/item/misc_weapons.dm:1977]:
```suggestion
		T.hitsound_charged = 'sound/weapons/cutter.ogg'
```
I don't want this sound being associated with a traitor weapon, as I made it specifically for the plasma cutter mining tool.

**Comment by flappybatpal** [comment]
From testing solo, the power of this item is in the defensive mode.

Three-five disarm attacks in barrier mode drain all 200 of a human's stamina and knock them down (depending on your luck with crits).
The resistance to disorientate effectively doubles the number of hits you need to take from a taser/baton to be stunned.
The armor stats are close to a security vest.
There is no way to stop the defensive mode short of chopping off the arm.

My assessment is that robust players would ignore the damage mode except against a stunned opponent and use the defensive mode alongside a one hand weapon or spam disarms where appropriate. The disarms can take players to 0 stamina in about twice the clicks of a baton.

File: code/obj/item/misc_weapons.dm (hunk 1 above)
Comment (CheffieGithub) [reply code/obj/item/misc_weapons.dm:1977]: sure can put something together but im no sound guy

**Comment by CheffieGithub** [comment]
that is a fair assessment though I think most "rampage" items are subordinate to stuns once the antag aquires them. Ill play around a tweak values. 
the plan was to make defensive mode drain a cell whenever it takes a hit from a projectile or weapon so it wasn't just perma raised stats so ill work on that

File: code/obj/item/misc_weapons.dm (hunk 1 above)
Comment (virvatuli-gh) [reply code/obj/item/misc_weapons.dm:1977]: if you can't come up with something suitable that's fine, this minor won't prevent the PR from being added to the game (there's far more important stuff to worry about). I can always make a sound for it assuming this PR passes other Dev's requirements.

**Comment by Tarmunora** [comment]
To recap what I said in admin channels about this:

It feels uninspired. At a glance, this looks like what you'd get if you put a c-saber, a barrier, and some bladed gloves into a blender and then strained out anything resembling counterplay or unique mechanics from the result

And secondly, I can't see this being used for anything other than as a "cut down the crew one by one after ganking security" rampage item, which, to be frank: meh.

**Comment by CheffieGithub** [comment]
That is a fair assement, what do you think can be changed to make it feel more unique and inspired? 
me and vispectra an idea to make it absorb projectiles with a shield which takes PU from a cell or to make defensive discharge the cell and offensive recharge it with delay then some kind of penalty when its fully drained. 
I may as well slap on input wanted this too.

**Comment by Tarmunora** [comment]
Overall, I'm of the opinion that we have more than enough traitor items that boil down to "item to beat people up with". Other devs might be more favorable to the concept and if one of them would like to sponsor this and help make it a good addition to the game, be my guest.

**Comment by Sord213** [comment]
I think this is kinda neat, you definitely put effort into balancing it which I appreciate. 

My two concerns:
1. It does not feel like a refitted mining tool at all. Gives off secret syndicate energy weapon which is fine but I'd rather it be clear what it is. Pick either advanced mining tool or syndicate special weapon.
2. The sprites are just...ok. They do not fit with our current style at all, imo.

I'd be fine with adding this if these two things are addressed.

**Comment by TDHooligan** [comment]
crossposting & summarising my forum post- i think this takes up two cool ideas that haven't really been filled (fire weapon, heavy shield) and does neither justice. I'd rather this gets some unique mechanic that turns it into one of the two.

**Comment by CheffieGithub** [comment]
Will get the sprites updated when we can since Vispectra is working on those

**Comment by CheffieGithub** [comment]
new sprites by shawtylikeamelody 
image

**Comment by deathrobotpunch** [comment]
Looks sck

**Comment by CheffieGithub** [comment]
Ready for review

**Comment by flappybatpal** [comment]
Runtimes when EMPd.

Attack sound is too mild IMO. While it's 30 damage it's not that noticeable or distinct.
The defensive mode doesn't feel OP now which is good.

Still overall makes me go "hmmm" for traitor value and lore/setting fit.

**Comment by CheffieGithub** [comment]
Fixed the EMP, working on a new sound and going to be porting behaviour from #13737 with permission to give it more utility which should increase its value for a traitor.

**Comment by flappybatpal** [comment]
I have consulted with the team and we have decided to decline this PR. We appreciate the effort put into making a new traitor item but don't feel the armblade provides a fresh enough experience for traitors. I'm sorry for the time it has taken to get to this response but ultimately there were not strong feelings in favor of the item.

**Pull request closed without merging**